com.vrcfury.vrcfury/Editor/VF/Feature/MoveMenuItemBuilder.cs

```diff
@@ -16,6 +16,15 @@ public class MoveMenuItemBuilder : FeatureBuilder<MoveMenuItem> {
             content.Add(VRCFuryEditorUtils.Info("This feature will move a menu item to another location. You can use slashes to make subfolders."));
             content.Add(VRCFuryEditorUtils.Prop(prop.FindPropertyRelative("fromPath"), "From Path"));
             content.Add(VRCFuryEditorUtils.Prop(prop.FindPropertyRelative("toPath"), "To Path"));
+            
+            var adv = new Foldout {
+                text = "Advanced Options",
+                value = false
+            };
+            content.Add(adv);
+            

```

Comment:
I feel like it does not need to be listed under "Advanced Options" as it could be seen to be trivial to allow it and not really too Advanced.